contracts/peg/PegStabilityModule.sol

```diff
@@ -187,119 +158,8 @@ contract PegStabilityModule is IPegStabilityModule, OracleRefV2 {
 
     /// ----------- Public View-Only API ----------
 
-    /// @notice calculate the amount of VOLT out for a given `amountIn` of underlying
-    /// First get oracle price of token
-    /// Then figure out how many dollars that amount in is worth by multiplying price * amount.
-    /// ensure decimals are normalized if on underlying they are not 18
-    /// @param amountIn amount of underlying token in
-    /// @return amountVoltOut the amount of Volt out
-    /// @dev reverts if price is out of allowed range
-    function getMintAmountOut(
-        uint256 amountIn
-    ) public view override returns (uint256 amountVoltOut) {
-        uint256 oraclePrice = readOracle();
-        _validatePriceRange(oraclePrice);
-
-        /// This was included to make sure that precision is retained when dividing
-        /// In the case where 1 USDC is deposited, which is 1e6, at the time of writing
-        /// the VOLT price is $1.05 so the price we retrieve from the oracle will be 1.05e6
-        /// VOLT contains 18 decimals, so when we perform the below calculation, it amounts to
-        /// 1e6 * 1e18 / 1.05e6 = 1e24 / 1.05e6 which lands us at around 0.95e17, which is 0.95
-        /// VOLT for 1 USDC which is consistent with the exchange rate
-        /// need to multiply by 1e18 before dividing because oracle price is scaled down by
-        /// -12 decimals in the case of USDC
-
-        /// DAI example:
-        /// amountIn = 1e18 (1 DAI)
-        /// oraclePrice = 1.05e18 ($1.05/Volt)
-        /// amountVoltOut = (amountIn * 1e18) / oraclePrice
-        /// = 9.523809524E17 Volt out
-        amountVoltOut = (amountIn * 1e18) / oraclePrice;
-    }
-
-    /// @notice calculate the amount of underlying out for a given `amountVoltIn` of Volt
-    /// First get oracle price of token
-    /// Then figure out how many dollars that amount in is worth by multiplying price * amount.
-    /// ensure decimals are normalized if on underlying they are not 18
-    /// @dev reverts if price is out of allowed range
-    function getRedeemAmountOut(
-        uint256 amountVoltIn
-    ) public view override returns (uint256 amountTokenOut) {
-        uint256 oraclePrice = readOracle();
-        _validatePriceRange(oraclePrice);
-
-        /// DAI Example:
-        /// decimals normalizer: 0
-        /// amountVoltIn = 1e18 (1 VOLT)
-        /// oraclePrice = 1.05e18 ($1.05/Volt)
-        /// amountTokenOut = oraclePrice * amountVoltIn / 1e18
-        /// = 1.05e18 DAI out
-
-        /// USDC Example:
-        /// decimals normalizer: -12
-        /// amountVoltIn = 1e18 (1 VOLT)
-        /// oraclePrice = 1.05e6 ($1.05/Volt)
-        /// amountTokenOut = oraclePrice * amountVoltIn / 1e18
-        /// = 1.05e6 USDC out
-        amountTokenOut = (oraclePrice * amountVoltIn) / 1e18;
-    }
-
     /// @notice returns the maximum amount of Volt that can be minted
-    function getMaxMintAmountOut() external view override returns (uint256) {
```

Comment:
thoughts on adding a getMaxRedeemAmountOut() that just reads balance() ?

Reply:
sure I can add. would you like to keep `getMaxMintAmountOut` in?

Reply:
algorithm ended up being `balance() * 1e18 / voltRate` to return amount of volt you can redeem